src/commands/script.rs

```diff
@@ -46,24 +49,28 @@ impl Script {
                 }
                 Err(_) => break,
             }
+            // Remove trailing whitespace
+            line = line.trim_end().to_string();
             if consuming_help {
-                if line.starts_with("# END HELP") {
+                if line.contains("END HELP") {
                     consuming_help = false;
-                } else if let Some(rest) = line.strip_prefix("# ") {
+                } else if line.contains("# ") || line.contains("// ") {
```

Comment:
TODO: generalize this to other common prefixes?

Reply:
are you suggesting we should support other comment prefixes (e.g. '!' or ';') as well?

I think that seems reasonable. I would prefer to not get into the business of capturing multi-line comments though, since it forces us to write some level of recursive parsing or escaping.